proptools/tag.go

```diff
@@ -23,13 +23,20 @@ import (
 // HasTag returns true if a StructField has a tag in the form `name:"foo,value"`.
 func HasTag(field reflect.StructField, name, value string) bool {
 	tag := field.Tag.Get(name)
-	for _, entry := range strings.Split(tag, ",") {
-		if entry == value {
-			return true
-		}
-	}
+	return !strings.Contains(value, ",") && hasTagRecursive(tag, value)
```

Comment:
What is the strings.Contains() here for?  It appears to silently fail any value that contains a `,`.  If that behavior is desirable it definitely deserves a test.